docs/README.md

```diff
@@ -64,13 +64,14 @@ You can download the Android MyAmbience app through the oficial Google Play Stor
 
 ## Requirements and Permissions
 
-The following permissions need to be set so that myAmbience can detect surrounding demonstrators:
+The following permissions need to be set so that MyAmbience can detect surrounding demonstrators:
 
-- Bluetooth on your device needs to be enabled and the app needs to have permissions for bluetooth
-- On Android devices location permissions need to be given in addition. We do not store location information, however this is required by the bluetooth functionality since on some Android versions these are coupled
+- Bluetooth on your device needs to be enabled and the app needs to have permissions for bluetooth.
+- On Android devices location permissions need to be given in addition. We do not store location information, however this is required by the bluetooth functionality since on some Android versions these are coupled.
+- In case you would like to get notifications when you download data from the demonstrators devices and export data, please allow notifications for MyAmbinece.
```

Comment:
```suggestion
- In case you would like to get notifications when you download data from the demonstrators devices and export data, please allow notifications for MyAmbience.
```